examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL/SDL.h>
```

Comment:
Ran `TOOLCHAIN=default PROJECT_DIR=examples/SDL2 ./jenkins.py --verbose` with `#include <SDL/SDL.h>` and it worked fine... I don't know what to say...